quickstarts/microsoft.databricks/databricks-workspace-with-managed-svc-customer-managed-keys/azuredeploy.json

```diff
@@ -0,0 +1,140 @@
+{
```

Comment:
why is azuredeploy written in JSON vs this other `nested_addAccessPolicy.bicep` written in bicep?